src/Kusto/Kusto/Commands/UpdateAzureRmKustoCluster.cs

```diff
@@ -81,11 +87,19 @@ public class UpdateAzureRmKustoCluster : KustoCmdletBase
 
         public override void ExecuteCmdlet()
         {
+
+            if (!string.IsNullOrEmpty(Tier) && string.IsNullOrEmpty(SkuName))
+            {
+                throw new ArgumentNullException("SkuName", "SkuName can not be null when Tier is defined");
```

Comment:
Please add parameter sets to do this validation - you should never have inline validation:
```
PS C:\> Update-AzureRmKustoCluster [-ResourceGroupName] <resourceGroupName> [-Name] <name> [-Location] <location> [[-SkuName] <skuName>] [[-Capacity] <capacity>]
PS C:\> Update-AzureRmKustoCluster -InputObject <ObjectType> [[-SkuName] <skuName>] [[-Capacity] <capacity>]
PS C:\> Update-AzureRmKustoCluster -ResourceId <string(resourceId of object)> [[-SkuName] <skuName>] [[-Capacity] <capacity>]
PS C:\> Update-AzureRmKustoCluster [-ResourceGroupName] <resourceGroupName> [-Name] <name> [-Location] <location> [-SkuName <skuName>] [-Tier <tier>] [[-Capacity] <capacity>]
PS C:\> Update-AzureRmKustoCluster -InputObject <ObjectType> [-SkuName <skuName>] [-Tier <tier>] [[-Capacity] <capacity>]
PS C:\> Update-AzureRmKustoCluster -ResourceId <string(resourceId of object)> [-SkuName <skuName>] [-Tier <tier>] [[-Capacity] <capacity>]
```

Reply:
After discussing abut the subject,we have decided to remove the Tier.
The current Tier is always Standard, and the clients can not set it.
Can I just remove it or it is too much of a breaking change?